delft3dworker/models.py

```diff
@@ -128,6 +128,38 @@ def _create_datafolder(self):
         if not os.path.exists(self.workingdir):
             os.makedirs(self.workingdir)
 
+    def export(self):
+        # Alternatives to this implementation are:
+        # - django-zip-view (sets mimetype and content-disposition)
+        # - django-filebrowser (filtering and more elegant browsing)
+
+        # from: http://stackoverflow.com/questions/67454/serving-dynamically-generated-zip-archives-in-django
+
+        zip_filename = 'export.zip'
+
+        # Open BytesIO to grab in-memory ZIP contents
+        # (be explicit about bytes)
+        stream = io.BytesIO()
+
+        # The zip compressor
+        zf = zipfile.ZipFile(stream, "w")
+
+        # Add files here.
+        # If you run out of memory you have 2 options:
+        # - stream
+        # - zip in a subprocess shell with zip
+        # - zip to temporary file
+        for root, dirs, files in os.walk(scene.workingdir):
+            for f in files:
+                if f.endswith('.png'):  # Could be dynamic or tuple of extensions
```

Comment:
Kan je niet beter https://docs.python.org/2/library/os.path.html#os.path.splitext gebruiken?
En zullen we gelijk ook checken voor jpg en gif?